 Fix NPE with void android.view.VelocityTracker.clear()

 AndroidRuntime: FATAL EXCEPTION: main
 AndroidRuntime: Process: com.android.settings, PID: 30816
 AndroidRuntime: java.lang.NullPointerException: Attempt to invoke virtual method 
 'void android.view.VelocityTracker.clear()' on a null object reference
 AndroidRuntime: 	at android.widget.ScrollView.onTouchEvent(ScrollView.java:692)

diff --git a/core/java/android/widget/ScrollView.java b/core/java/android/widget/ScrollView.java
index 00991d5..2ae12fb 100644
--- a/core/java/android/widget/ScrollView.java
+++ b/core/java/android/widget/ScrollView.java
@@ -690,7 +690,9 @@ public boolean onTouchEvent(MotionEvent ev) {
                     if (overScrollBy(0, deltaY, 0, mScrollY, 0, range, 0, mOverscrollDistance, true)
                             && !hasNestedScrollingParent()) {
                         // Break our velocity if we hit a scroll barrier.
-                        mVelocityTracker.clear();
+                        if (mVelocityTracker != null) {
+                            mVelocityTracker.clear();
+                        }
                     }
 
                     final int scrolledDeltaY = mScrollY - oldY;
